source/backend/tests/unit/api/Services/DocumentFileServiceTest.cs

```diff
@@ -109,6 +109,37 @@ public void GetPimsDocumentTypes_Acquisition_Success()
             acquisitionFileDocumentRepository.Verify(x => x.GetAllByAcquisitionFile(It.IsAny<long>()), Times.Once);
         }
 
+        [Fact]
+        public void GetPimsDocumentTypes_Project_Success()
```

Comment:
why getPimsDocumentTypes? shouldn't this be GetAllProjectDocuments?

Reply:
Updated